src/lib.rs

```diff
@@ -67,22 +76,24 @@
 //! ### Transaction Execution
 //!
 //! To start a VM on the Transaction level, use the `TransactionVM`
-//! struct. Usually, you use the sequencial memory module, which we
-//! have a definition called `SeqTransactionVM`. Call to
-//! `TransactionVM::new` would require the transaction to be valid
-//! (according to the rules for an Ethereum transaction). For an
-//! invalid transaction, it is possible for the VM to panic. If you
-//! want to handle untrusted transactions, you should use
-//! `SeqTransactionVM::new_untrusted` which will return an error if
-//! the transaction is invalid.
+//! struct. Usually, you want to use the sequential memory module
+//! which can be done using the type definition
```

Comment:
please a cambridge comma:

```
Usually, you'll want to use the sequential memory module, which can be done...
```

Reply:
@whilei  I'm not totally certain what you mean. Do you mean to _remove_ the comma. Perhaps it's simpler to just go in and make the change yourself as it will remove back-and-forth. Thanks.

Reply:
:) 🖊 👓 💉